bugwarrior/db.py

```diff
@@ -25,8 +26,9 @@
 MARKUP = "(bw)"
 
 
-DOGPILE_CACHE_PATH = os.path.expanduser(
-    os.getenv('XDG_CACHE_HOME', '~/.cache') + '/dagd.dbm')
+PYVER = '%i.%i' % sys.version_info[:2]
+DOGPILE_CACHE_PATH = os.path.expanduser(''.join([
+    os.getenv('XDG_CACHE_HOME', '~/.cache'), '/dagd-py', PYVER, '.dbm']))
```

Comment:
The actual database format is different? That doesn't make much sense to me.

Or is it just a bytes-vs-strings issue where we could just convert bytes to unicode in python3?